lib/registries/addon/components/registries-metadata/template.hbs

```diff
@@ -91,15 +91,25 @@
         <div local-class='Field'>
             <EditableField
                 data-analytics-scope='Affiliated institutions'
+                @managerComponent={{component 'editable-field/institutions-manager' node=this.registration}}
                 @registration={{this.registration}}
                 @title={{t 'registries.registration_metadata.affiliated_institutions'}}
-            as |editable|>
-                <InstitutionsWidget
-                    data-analytics-scope='Registries overview'
-                    @node={{this.registration}}
-                    @shouldShowModal={{editable.shouldShowEditable}}
-                    @hideModal={{editable.hideEditable}}
-                />
+                @name='affiliated institutions'
+                as |field|
+            >
+                <field.edit>
+                    <InstitutionSelectList
+                        @user={{this.currentUser.user}}
+                        @node={{this.registration}}
+                        @manager={{field.manager}}
+                    />
+                </field.edit>
+                <field.display>
+                    <InstitutionsList
+                        @reloadList={{action (mut field.manager.reloadList)}}
```

Comment:
We should avoid directly mutating properties on the manager -- let it manage itself. Maybe we could yield a `bindReload` action instead of `reloadList` itself?

```hbs
{{! editable-field/template.hbs }}
{{yield
  ...
  bindReload=(action (mut this.reloadList))
}}
```
```hbs
<InstitutionsList
  @bindReload={{field.manager.bindReload}}
```

Though ideally, the manager would manage state for `<InstitutionsList>` too, so we wouldn't use the "bind actions up" pattern at all. Maybe let's think of that as a future improvement once the manager pattern is more settled.